Xamarin.Forms.Core/Slider.cs

```diff
@@ -65,18 +63,30 @@ public Slider(double min, double max, double val) : this()
 			Value = val.Clamp(min, max);
 		}
 
+		/// <summary>
+		/// Gets or sets the highest possible Value for the slider.
+		/// Must be data-bound before Value to avoid Value being set to default Maximum if Value is greater than default Maximum.
+		/// </summary>
```

Comment:
There are many summary comments in the source code. Just search for it in the solution and you'll get a list. Do you still want me to remove mine?